src/aiortc/rtcdtlstransport.py

```diff
@@ -48,6 +48,10 @@
 
 assert lib.OpenSSL_version_num() >= 0x10002000, "OpenSSL 1.0.2 or better is required"
 
+# Log TLS secrets to a file, similar to browsers
+SSLKEYLOGFILE = os.getenv('SSLKEYLOGFILE')
+if SSLKEYLOGFILE:
+    logger.warning('Logging all TLS keys to %s', SSLKEYLOGFILE)
```

Comment:
Why are we emitting a warning here?

Reply:
Because logging all encryption keys is something you probably don't want by default, so I thought it might be a good idea to log a warning if it is enabled.